spec/requests/users_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users/'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/1'
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [ ] Good job working on your controller specs. However, you seem to only have handled one end of the equation. Kindly include tests that ensures correct view template is rendered. Example : expect(response).to render_template(:show) 
- [ ] Kindly also include tests that ensures the response body includes the correct placeholder text. Example : expect(response.body).to include("Welcome to user index page")

Reply:
@Nemwel-Boniface Hello mate yes, 
i did forget to implement this requirement:
- If the response body includes correct placeholder text.

but the template test is already done within the: `spec/view` directory: 
![image](https://user-images.githubusercontent.com/99597333/204902069-9ba7580e-fc21-4eac-955b-41145d3373c2.png)